src/plugins/presentation_util/public/components/controls/control_group/editor/manage_control.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useEffect, useState } from 'react';
+import {
+  EuiFlyoutHeader,
+  EuiButtonGroup,
+  EuiFlyoutBody,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiFieldText,
+  EuiFlyoutFooter,
+  EuiButton,
+  EuiFormRow,
+  EuiForm,
+  EuiButtonEmpty,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { ControlGroupStrings } from '../control_group_strings';
+import { ControlEditorComponent, ControlWidth } from '../../types';
+import { CONTROL_WIDTH_OPTIONS } from '../control_group_constants';
+
+interface ManageControlProps {
+  title?: string;
+  onSave: () => void;
+  width: ControlWidth;
+  onCancel: () => void;
+  removeControl?: () => void;
+  controlEditorComponent?: ControlEditorComponent;
+  updateTitle: (title: string) => void;
+  updateWidth: (newWidth: ControlWidth) => void;
+}
+
+export const ManageControlComponent = ({
+  controlEditorComponent,
+  removeControl,
+  updateTitle,
+  updateWidth,
+  onCancel,
+  onSave,
+  title,
+  width,
+}: ManageControlProps) => {
+  const [currentTitle, setCurrentTitle] = useState(title);
+  const [currentWidth, setCurrentWidth] = useState(width);
+
+  const [controlEditorValid, setControlEditorValid] = useState(false);
+  const [editorValid, setEditorValid] = useState(false);
+
+  useEffect(() => setEditorValid(Boolean(currentTitle)), [currentTitle]);
+
+  return (
+    <>
+      <EuiFlyoutHeader hasBorder>
+        <EuiTitle size="m">
+          <h2 id="flyoutTitle">{ControlGroupStrings.manageControl.getFlyoutTitle()}</h2>
```

Comment:
you'll wanna update this id!

Reply:
Pretty sure these ids are not needed, serves me right for copying directly from EUI examples hah